enocean/protocol/EEP.xml

```diff
@@ -1389,20 +1389,12 @@
               <min>0</min>
               <max>255</max>
             </range>
-            <scale>
```

Comment:
Sorry,
Just read the code of the EnOcean library and it seems that once `range` is defined `scale` must be defined as well and vice-versa.
So we should just add:
```xml
            <scale>
              <min>0</min>
              <max>255</max>
            </scale>
```